recipes/deepspeed/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "deepspeed" %}
+{% set version = "0.6.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/deepspeed-{{ version }}.tar.gz
+  sha256: ad2de46d6609798fa3525ef8eb0e1ddfb66e06983b0af967b462b1511f21cc9c
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  noarch: python
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - pytorch
+  run:
+    - hjson-py
+    - ninja
```

Comment:
This ninja isn't the PyPI one from https://pypi.org/project/ninja/. See issue at https://github.com/conda-forge/ninja-feedstock/issues/26. Need to have a `python-ninja` package or similar on conda-forge first.